datadog_checks_dev/datadog_checks/dev/tooling/commands/dep.py

```diff
@@ -238,13 +242,19 @@ def is_version_compatible(marker, supported_versions):
 @click.option('--sync', '-s', is_flag=True, help='Update the `agent_requirements.in` file')
 @click.option(
     '--check-python-classifiers',
-    '-s',
+    '-c',
     is_flag=True,
     help="""Only flag a dependency as needing an update if the newest version has python classifiers matching the marker.
     NOTE: Some packages may not have proper classifiers.""",
 )
+@click.option('--include-security-deps', '-i', is_flag=True, help="Attempt to update security dependencies")
```

Comment:
Do you need to specify a default value for this?

Reply:
No, since it is a flag the default is `False` when the flag is not used